### ISSUE TITLE
Is it possible to delete recordings?

### ISSUE THREAD

**Issue by parvez** [davglass/doorbot#17]
Can we have delete api in the module?

**Comment by davglass** [comment]
I'll have to check to see if they have an API for it. I've never needed to delete a recording, I didn't even know you could :)

**Comment by parvez** [comment]
Not sure if this will be helpful. The ring website calls these apis. I think the API you use are different.

curl 'https://ring.com/account/activity/delete_dings' -X DELETE <PARAMS>
curl 'https://ring.com/account/activity/fetch_dings?limit=50&older_than=6000000996460054000' 
curl 'https://ring.com/account/activity/ding_video?doorbot_id=DOORBOT_ID&ding_id=DING_ID'

**Comment by davglass** [comment]
Yes, those API's are not the same. I would need to reverse engineer the current API's to handle it. I just haven't had a need for them as of yet.
